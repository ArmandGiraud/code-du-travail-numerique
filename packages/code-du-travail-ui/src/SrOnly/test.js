import React from "react";
import { render } from "react-testing-library";
import SrOnly from ".";

describe("<SrOnly />", () => {
  test("should render", () => {
    const { container } = render(<SrOnly />);
    expect(container).toMatchSnapshot();
  });
});
