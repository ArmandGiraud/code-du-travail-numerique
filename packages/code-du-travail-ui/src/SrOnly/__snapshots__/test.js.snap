// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SrOnly /> should render 1`] = `
.c0 {
  position: absolute;
  width: 1px;
  height: 1px;
  margin: -1px;
  padding: 0;
  overflow: hidden;
  -webkit-clip: rect(0,0,0,0);
  clip: rect(0,0,0,0);
  border: 0;
}

<div>
  <div
    class="c0"
  />
</div>
`;
